### PR TITLE
Add touch events fixes and e2e tests

### PR DESCRIPTION
**Pull request by apajaro**

### CODE REVIEW

**Comment by mauroporras** [comment]
Looks like a WIP. Converted to draft.

File: package.json [hunk 1]
```diff
@@ -57,7 +57,8 @@
   },
   "dependencies": {
     "@zeainc/zea-engine": "2.0.0",
-    "dom-to-image": "^2.6.0"
+    "dom-to-image": "^2.6.0",
+    "npm-run-all": "^4.1.5"
```
Comment (mauroporras) [inline package.json:61]: This one needs to be a dev dep.

**Review by mauroporras — CHANGES_REQUESTED** [review]
Let's fix conflicts first.

**Comment by apajaro** [comment]
There's an issue with iOS devices that doesn't allow selection of the right geometries on Selection tool.
This has to do with the changes introduced in the `StandardSurfaceGeomDataShader` class for `UInt8 buffer`.

**Comment by phtaylor** [comment]
> There's an issue with iOS devices that doesn't allow selection of the right geometries on Selection tool.
> This has to do with the changes introduced in the `StandardSurfaceGeomDataShader` class for `UInt8 buffer`.
> 
> @phtaylor

this is fixed now. Lets get the conflicts resolved and merge this as soon as possible.

**Comment by phtaylor** [comment]
Note: I changed the handle size, so this may effect the e2e tests. I'm not sure why the appeared so huge for me.

**Review by phtaylor — APPROVED** [review]

**Pull request closed without merging**